src/js/components/container/FormContainer.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import ReactDOM from "react-dom";
+import Input from "../presentational/Input.jsx";
+
+class FormContainer extends Component{
+    constructor(){
```

Comment:
do you not need to pass in props here?

Reply:
Thanks for catching! Current master branch addresses this. I'll go ahead and resolve this issue and close the PR.